openedx_external_enrollments/management/commands/migrate_other_course_settings.py

```diff
@@ -0,0 +1,29 @@
+"""
+Script for exporting all courses from Mongo to a othercoursesettings table on edxapp database.
+"""
+from django.core.management.base import BaseCommand
+
+from openedx_external_enrollments.edxapp_wrapper.get_course_settings import migrate_course_settings
+
+
+class Command(BaseCommand):
+    """
+    Migrate all courses via django command.
+    """
+    help = 'Export all courses from mongo to migrate into OtherCourseSettings table.'
```

Comment:
Please add an example of the command and a description of the allowed parameters. @ivanvgh

Reply:
Done.

Reply:
Thanks for adding them, can you add them to the command help output? @ivanvgh